pnpm-workspace.yaml

```
@@ -7,4 +7,11 @@ packages:
 catalog:
   drizzle-orm: ^0.38.3
-  drizzle-kit: ^0.30.1
\ No newline at end of file
+  drizzle-kit: ^0.30.1
+  "@dotenvx/dotenvx": ^1.33.0
+  typescript: ^5.7.3
+  vite: ^6.0.7
+  vite-tsconfig-paths: ^4.3.2
+  vite-plugin-dts: ^4.4.0
+  vite-dts: ^4.4.0
+  vitest: ^2.1.8
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)